src/app/base/components/DhcpFormFields/DhcpFormFields.test.tsx

```diff
@@ -9,24 +10,51 @@ import { Labels } from "./DhcpFormFields";
 
 import DhcpForm from "app/base/components/DhcpForm";
 import type { RootState } from "app/store/root/types";
+import { NodeStatus, NodeStatusCode } from "app/store/types/node";
 import {
   controllerState as controllerStateFactory,
   deviceState as deviceStateFactory,
   dhcpSnippet as dhcpSnippetFactory,
   dhcpSnippetState as dhcpSnippetStateFactory,
   machine as machineFactory,
   machineState as machineStateFactory,
+  machineStateList as machineStateListFactory,
+  machineStateListGroup as machineStateListGroupFactory,
   subnet as subnetFactory,
   subnetState as subnetStateFactory,
   rootState as rootStateFactory,
 } from "testing/factories";
 
 const mockStore = configureStore();
-
+const machines = [
+  machineFactory({
```

Comment:
Do these tests need all these specific props defined? If not it'd probably be clearer to just define the ones we need (if any).